src/main/java/de/digitalcollections/iiif/hymir/frontend/ExceptionAdvice.java

```diff
@@ -34,6 +39,18 @@ public void handleResourceNotFoundException(Exception exception) {
     // NOP
   }
 
+  /**
+   * Sometimes scaling an image can lead to broken dimensions (usually because of rounding to 0).
+   * @param exception The exception to handle
+   * @return An error description for the client
+   */
+  @ExceptionHandler(ScalingException.class)
+  @ResponseStatus(HttpStatus.BAD_REQUEST)
+  @ResponseBody
+  protected ApiError handleSolrException(ScalingException exception) {
```

Comment:
Das sollte wohl besser
```suggestion
  protected ApiError handleScalingException(ScalingException exception) {
```
heißen, oder?